### DISCUSSION TITLE
Grub2win problem with ChromeOS

### DISCUSSION THREAD

**Discussion by Paul-Chi** [sebanc/brunch#1753] [Q&A]
I recently Installed Chrome OS using wsl2 Ubuntu... It Installed perfectly, but Grub2win tells me it doesn't support Chrome OS is there a way to resolve this problem?

**Comment by ibnsamy96** [comment]
Hi Paul, Did you find a solution?

**Comment by Paul-Chi** [comment]
It's working when I use the multi_install.sh file with Parrot OS .... using
the Submenu option in Grub2Win
